controllers/subscriptions.go

```diff
@@ -21,7 +22,6 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/sirupsen/logrus"
-	"golang.org/x/exp/slices"
```

Comment:
this was moved to core package `slices` in go 1.21

we are now on go 1.22 in entitlements so we can update this to not pull from the experimental package anymore :partying_face: